components/briefing/macros.hpp

```diff
@@ -1 +1,9 @@
-#include "../../macros.hpp"
\ No newline at end of file
+#include "../../macros.hpp"
+
+#define RESPAWN_MODE_NAME_TIMED "Timed"
```

Comment:
Prefer to have these in `respawn_macros.hpp` so they're in the same place as the things they refer to.